tests/conftest.py

```diff
@@ -1372,3 +1383,27 @@ def local_module_package_index(modules_v2_dir: str) -> Iterator[str]:
             ModuleTool().build(path=path, output_dir=artifact_dir)
         dir2pi(argv=["dir2pi", artifact_dir])
         yield os.path.join(artifact_dir, "simple")
+
+
+@pytest.fixture
+async def migrate_db_from(
+    request: pytest.FixtureRequest, hard_clean_db, hard_clean_db_post, postgresql_client: asyncpg.Connection, server_pre_start
+) -> AsyncIterator[Callable[[], Awaitable[None]]]:
```

Comment:
Would it make sense to package this onto a conf.py in the db folder (so that this doesn't end up in pytest-extensions?)

Or if it should be in pytest extensions, would it not make sense to move the `PGRestore` to this level?

Reply:
Is this comment still relevant knowing `db.common` is packaged into `pytest_inmanta_extensions`?